app/models/package.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Package < ApplicationRecord
+  enum status: {
+    pending: 0,
+    paid: 1,
+  }
+
```

Comment:
I think you also need to add the money helpers and add some validations into the `package` model like this. AFter adding these you might need to update your test specs also:

```ruby
monetize :price_cents, allow_nil: false, numericality: { greater_than_or_equal_to: 0 }
monetize :retail_price_cents, allow_nil: false, numericality: { greater_than_or_equal_to: 0 }

```